<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDocument</class>
 <widget class="QDialog" name="DialogDocument">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNuevo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Nodo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar Nodo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPDF">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGrafico">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Gráfico</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardarNodo">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSeleccion">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Selección Actual</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnParrafo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Párrafo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVineta">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Viñeta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImagen">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Imagen</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>381</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblTitulo">
      <property name="text">
       <string>Título (Opcional)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtTitulo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lblContenido">
      <property name="text">
       <string>Contenido (Párrafo)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="edtContenido"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>381</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="lblImagen">
      <property name="text">
       <string>Imagen</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="edtImagen"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnChoose">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
